km/km_fork.c

```diff
@@ -163,7 +163,8 @@ static void km_fork_remove_parent_vmstate(void)
    // Should try to free all of the stacks for the now defunt vcpu threads?
    for (int i = 0; i < machine.vm_vcpu_cnt; i++) {
       if (machine.vm_vcpus[i] != NULL) {
-         km_vcpu_fini(machine.vm_vcpus[i], 0);
+         machine.vm_vcpus[i]->vcpu_thread = 0;   // that thread was in parent
```

Comment:
Why do you do this?
km_vcpu_fini() sets machine.vm_vcpus[i] to NULL so vcpu_thread will not even exist shortly.

Reply:
I see what is going on.  km_vcpu_fini() will try to signal an invalid threadid when we are in the child process of a fork.
bleh.